manifests/karina-operator.yaml

```diff
@@ -110,20 +110,20 @@ spec:
         control-plane: karina-operator
     spec:
       containers:
-        - args:
-            - operator
-            - --enable-leader-election
-            - --log-level=debug
-          command:
-            - /bin/karina
-          image: docker.io/flanksource/karina:{{.karinaOperator.version}}
-          imagePullPolicy: IfNotPresent
-          name: karina-operator
-          resources:
-            limits:
-              cpu: 500m
-              memory: 256Mi
-            requests:
-              cpu: 100m
-              memory: 128Mi
-      serviceAccountName: manager
+      - args:
+        - operator
+        - --enable-leader-election
+        - --log-level=debug
+        command:
+        - /bin/karina
+        image: docker.io/flanksource/karina:latest
```

Comment:
I think this may unintended - Could we remove all the changes not specifically related to this PR ?

Reply:
I can. Or I can correct the template for this file. I was tried to move the `karina-operator` upstream to the new folder structure we just introduced 2 weeks ago (using directory as upstream name instead of a single yaml file)

The changes are:
- `manifests/karina-operator.yaml` has been move to `manifest/karina-operator/base/kustomization.yaml`
- `manifests/crd/karina-operator.yaml` has been move to `manifest/karina-operator/crd/kustomization.yaml`

Reply:
I pushed the fix